### ISSUE TITLE
KotlinNullPointerException: getBrowserToolbarView()

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/fenix#14289]
https://sentry.prod.mozaws.net/operations/firefox-nightly/issues/9230976/?referrer=github_plugin

```
KotlinNullPointerException: None
    at org.mozilla.fenix.browser.BaseBrowserFragment.getBrowserToolbarView(BaseBrowserFragment.kt:1)
    at org.mozilla.fenix.browser.BaseBrowserFragment.onAccessibilityStateChanged(BaseBrowserFragment.kt:1)
...
(7 additional frame(s) were not displayed)
```

**Comment by pocmo** [comment]
Top crash in Nightly on Sentry.

**Comment by andreicristianpetcu** [comment]
Is this a startup crash? I'm getting a startup crash and I'm not sure if I should report it.

`adb shell pidof -s org.mozilla.fenix.nightly` returns nothing

**Comment by mcarare** [comment]
Although I cannot reproduce this, I have submitted a speculative fix for this. I am still unsure why the `browserToolbarView` is null, because in https://github.com/mozilla-mobile/fenix/pull/13880 I tried to add the a11y listener after the init of the `browserToolbarView`.
Also, I wonder if the use of a private var and a getter that throws `KotlinNullPointerException` is better than using `lateinit`.

There are other crashes with`KotlinNullPointerException`, with calls to `getBrowserToolbarView `: 

https://sentry.prod.mozaws.net/operations/fenix/issues/8277631/

@pocmo, any thoughts?

**Comment by mcarare** [comment]
Closing this, as Sentry does not show any new crashes for Fenix Nightly versions that include the fix.
